Example/android/gradle.properties

```diff
@@ -18,3 +18,4 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+android.enableAapt2=false
```

Comment:
This is no longer required as of RN 57 right?

Reply:
This proyect has react-native 0.44 as declared on the package.json

Reply:
Ah, I see.  Setting this potentially breaks other stuff if someone were to follow it.  If it works as is here in the example, it's fine.  But would be better if we could peg it to 0.57.x and remove this line.

Reply:
Someone should update the example to RN 0.57

Reply:
True dat, i'll open up a separate issue.